cumulusci/core/config/BaseProjectConfig.py

```diff
@@ -614,3 +608,20 @@ def process_github_dependency(self, dependency, indent=None):
             repo_dependencies.extend(unpackaged_post)
 
         return repo_dependencies
+
+    def _find_release_version(self, repo, indent, install_latest=None):
```

Comment:
the arg name is a little confusing I think, install_latest installs the latest pre-release, whereas install_latest=False installs the release at `/latest`...the latest release-release. maybe `include_prereleases`?